gtests/net/mptcp/add_addr/add_addr_client.pkt

```diff
@@ -21,4 +21,4 @@
 +0.0    < . 1:1(0) ack 101 win 256 <nop,nop,TS val 705 ecr 305,add_address addr[saddr] hmac=auto,dss dack8=101 dll=0 nocs>
 
 +0.4 close(3) = 0
-+0.0   > F. 101:101(0) ack 1 <nop, nop,TS val 494 ecr 700,dss dack4=1 dsn8=101 ssn=0 dll=1 nocs fin, nop, nop>
++0.0   > . 101:101(0) ack 1 <nop, nop,TS val 494 ecr 700,dss dack4=1 dsn8=101 ssn=0 dll=1 nocs fin, nop, nop>
```

Comment:
should we add another packet with the `> F.` to always check we have the FIN, not just the DATA+FIN?
(or maybe we have other tests validating that already? or should we validate that in specific DATA_FIN tests?)